iib/web/models.py

```diff
@@ -923,6 +925,7 @@ def from_json(cls, kwargs, batch=None):
                 'bundles',
                 'distribution_scope',
                 'deprecation_list',
+                'registry_auths',
```

Comment:
Is this needed? You're already specifying it in `RequestIndexImageMixin._from_json`. I think we only need to specify it in one place. If this is only to be used for `add` requests, and not `rm`, then I think it should only be listed here and not in `RequestIndexImageMixin._from_json`.